endpoints/bookstore-grpc/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM debian:jessie
```

Comment:
Our other endpoints sample uses the [python runtime](https://github.com/GoogleCloudPlatform/python-runtime), any strong reason for not using it here?